services/backup_socrata.py

```diff
@@ -13,13 +14,16 @@
 AWS_ACCESS_ID = os.getenv("AWS_ACCESS_ID")
 AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
 SOCRATA_APP_TOKEN = os.getenv("SOCRATA_APP_TOKEN")
+SOCRATA_API_KEY_ID = os.getenv("SOCRATA_API_KEY_ID")
+SOCRATA_API_KEY_SECRET = os.getenv("SOCRATA_API_KEY_SECRET")
 
 
 def export_dataset(resource_id):
-    url = f"https://data.austintexas.gov/resource/{resource_id}.csv?$limit=99999999&$$app_token={SOCRATA_APP_TOKEN}"
-    res = requests.get(url)
+    url = f"https://datahub.austintexas.gov/resource/{resource_id}.csv?$limit=99999999&$$app_token={SOCRATA_APP_TOKEN}"
+    basic = HTTPBasicAuth(username=SOCRATA_API_KEY_ID, password=SOCRATA_API_KEY_SECRET)
+    res = requests.get(url, auth=basic)
```

Comment:
might be good to add a timeout here? i think that is something baked into sodapy so we don't often worry about it...

Reply:
hmmm yes, I was also thinking about adding some pagination in case we want to back up some really large datasets.

Reply:
ok added a timeout and a potentially circuitous approach to use pagination!